docs/user-guide.md

```diff
@@ -30,20 +30,21 @@ helm install akri akri-helm-charts/akri
 
 To use the latest containers of the Akri components, add `--set useLatestContainers=true` when installing Akri like so:
 ```sh
-helm install akri akri-helm-charts/akri \
+helm install akri akri-helm-charts/akri-dev \
    --set useLatestContainers=true 
 ```
 
+Starting in v0.4.0, Akri will support two Agents, one that includes all supported Discovery Handlers and one without any
```

Comment:
It sounds like 2 agents will be running while we support the agent to run in 2 different ways/modes,

Reply:
I can see how it could be read that way. Would "Akri will support two Agent builds, one that includes all supported Discovery Handlers and one without any... By default, our Helm chart deploys an Agent with the latter build" make more sense?

Reply:
Elaborated on how we are switching to a new default and why here: afd6d976f8124bc7

Reply:
This looks great. Thanks Kate.